src/stores/api/types/eth1.go

```diff
@@ -28,6 +28,19 @@ type SignHexPayloadRequest struct {
 	Data hexutil.Bytes `json:"data" validate:"required" example:"0xfeee" swaggertype:"string"`
 }
 
+type SignMessagePayloadRequest struct {
+	Data hexutil.Bytes `json:"message" validate:"required" example:"0xabcd1234" swaggertype:"string"`
+}
+
+type SignMessagePayloadResponse struct {
+	Message     string `json:"message" example:"any message goes here" swaggertype:"string"`
```

Comment:
Message example is incorrect since we expect message to be passed as hex value

Reply:
As I understand it the message should be a human readable message

Reply:
But the `message` will correspond to what the user provides via API therefore it will be hex value:
```
type SignMessagePayloadRequest struct {
	Data hexutil.Bytes `json:"message" validate:"required" example:"0xabcd1234" swaggertype:"string"`
```

Reply:
Yes, thats true if we follow the input spec but I was wondering if that input :
type SignMessagePayloadRequest struct {
	Data string `json:"message" validate:"required" example:"any text message" swaggertype:"string"`
}
would not be more appropriate 🤔 . I really cannot see the point of signing a non readable message with that endpoint.